packages/splits-sdk-react/src/hooks/splitV1.ts

```diff
@@ -113,6 +113,12 @@ export const useCreateSplit = (): {
         setError(undefined)
         setTxHash(undefined)
 
+        const chainId = splitsClient._walletClient?.chain.id
```

Comment:
do we only need these changes in splitV1?

Reply:
yes, only splitmain has the different abis depending on chain (mainnet vs all others). that's why we need the chain id when getting the events. don't need to do that for any other contract